mono/mini/interp/interp.c

```diff
@@ -229,7 +229,9 @@ int mono_interp_traceopt = 0;
 #define MINT_IN_DEFAULT default:
 #endif
 
-static void
+static
+MONO_NEVER_INLINE // Inling this causes caller to use more stack.
```

Comment:
single line if possible

Reply:
```
static MONO_NEVER_INLINE void
foo
```
ok, I can search for ^foo.


```
static MONO_NEVER_INLINE void foo
```
less ok but still ok